### PR TITLE
Login update

### PR DESCRIPTION
**Pull request by JimmyWxzzz**

### CODE REVIEW

File: src/main/java/com/sweepy/urlService/urlService.java [hunk 1]
```diff
@@ -1,7 +1,6 @@
 package com.sweepy.urlService;
 
 import javax.servlet.http.HttpServletResponse;
-import javax.swing.*;
 import java.io.IOException;
 
 public interface urlService {
```
Comment (LeonYanc) [inline src/main/java/com/sweepy/urlService/urlService.java]: The class name should always start with capital letter, only function/variable starts with lower case. Please change for all the ones applicable

**Comment by LeonYanc** [comment]
@zhangxyleon pls also review

**Comment by JimmyWxzzz** [comment]
image
JWT method login

File: src/main/java/com/sweepy/urlService/urlService.java (hunk 1 above)
Comment (JimmyWxzzz) [reply src/main/java/com/sweepy/urlService/urlService.java]: addressed

**Review by LeonYanc — APPROVED** [review]

**Pull request merged**